phpBB/assets/javascript/editor.js

```diff
@@ -226,6 +230,61 @@ function addquote(post_id, username, l_wrote) {
 	return;
```

Comment:
Oh right, I removed the other one at the end of bbfontstyle() :sweat:

I removed the right one this time. I didn't put back the one from bbfontstyle() because it felt stupid doing so.